homeassistant/components/purpleair/sensor.py

```diff
@@ -172,7 +166,7 @@ class PurpleAirSensorEntityDescription(
     ),
     PurpleAirSensorEntityDescription(
         key="voc",
-        name="VOC",
+        translation_key="voc",
```

Comment:
This has a device class which is `aqi` but it's name is `voc`?

Reply:
Sorry, completely missed it was discussed above but should we not remove the device class in this case then?

Reply:
Yes, I am going to do that :)